### PR TITLE
Update 2018 07 30

### PR DESCRIPTION
**Pull request by bravely**
Just updating with the newest changes from the main `plug_statsd` repo so we can get Statix support.

Only code I added/changed was moving the `ExStatsBackend.histogram` method into the elixir 1.5 check statement, and adding histogram to the StatixBackend.

### CODE REVIEW

File: lib/plug_statsd/statix_backend.ex [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+      timing(name, elapsed, 1.0)
+    end
+
+    def timing(name, elapsed, rate) when is_float(rate) do
+      get_conn().timing(name, elapsed, sample_rate: rate)
+    end
+
+    defp get_conn do
+      Application.get_env(:plug_statsd, :statix_backend_conn)
+    end
```
Comment (terrence2) [inline lib/plug_statsd/statix_backend.ex:27]: I don't see `histogram` in this backend?

**Review by terrence2 — APPROVED** [review]
It doesn't look like the histogram function got added to the statix backend? Otherwise, this looks awesome! Thanks for upgrading us!

File: lib/plug_statsd/statix_backend.ex (hunk 1 above)
Comment (bravely) [reply lib/plug_statsd/statix_backend.ex:27]: whoops apparently I saved that after I committed and pushed

it's up now

**Review by Shayon — APPROVED** [review]
For future I would prefer that some of these changes were split into separate PR's to help tell a distinct story about the change.  

I see a bunch of other people's commits, presumably from the original fork.  It would have been nice, if loading those in was a single PR and then the changes you made on top be another.

**Comment by bravely** [comment]
so github won't let me react to your post Shayon so I'll do it here

👍

**Pull request merged**